masterlist.yaml

```diff
@@ -2778,35 +2778,110 @@ plugins:
       - Delev
       - Relev
 
+  - name: '(EBMD(_AllInOne|-UF04P Patch)|Even(BetterMDesc|BMD_(Automatron|FarHarbor|NukaWorld)DLC))\.esp'
+    url:
+      - link: 'https://www.nexusmods.com/fallout4/mods/36105/'
+        name: 'Even Better Mod Descriptions'
+      - link: 'https://www.nexusmods.com/fallout4/mods/74459/'
+        name: 'Illusive Fixes and Merges'
   - name: 'EBMD_AllInOne.esp'
+    msg:
+      - <<: *alsoUseX
+        subs: [ '[Even Better Mod Descriptions Fixed](https://www.nexusmods.com/fallout4/mods/74459/)' ]
+        condition: 'not version("EBMD_AllInOne.esp", "4.1", =)'
```

Comment:
The available comparators are: [Link](https://loot-api.readthedocs.io/en/stable/metadata/conditions.html#types)
A single `=` is not a valid comparator.